### PR TITLE
Denis Doubinin Md sa2 17 21

### PR DESCRIPTION
**Pull request by DenisDoubinin**
Some homeworks

### CODE REVIEW

File: Denis_Doubinin/02.Git.Local/readme.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Git code
+
+
+   96  git init
```
Comment (pluhin) [inline Denis_Doubinin/02.Git.Local/readme.md:4]: Please use git markdown for reports (for code/log sections too)

File: Denis_Doubinin/03.Git.Hosting/readme.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+Push has started in the origin_gitlab. Branch main
+client_global_hostkeys_private_confirm: server gave bad signature for RSA key 0
+Everything up-to-date
+Push to origin_gitlab branch main completed successfully
+Updated 0 paths from the index
+
+Beginin error
+
+Push has started in the origin_bitbucket. Branch dev1.txt
+error: src refspec dev1.txt does not match any
```
Comment (pluhin) [inline Denis_Doubinin/03.Git.Hosting/readme.md:113]: You have too many error messages in output of your script, please check

**Review by pluhin — CHANGES_REQUESTED** [review]

**Review by pluhin — COMMENTED** [review]
Please go trough all your reports, and add git markdown section to your code sections

It is not readable by webui, example:

image

**Comment by pluhin** [comment]
https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

File: Denis_Doubinin/02.Git.Local/readme.md (hunk 1 above)
Comment (DenisDoubinin) [reply Denis_Doubinin/02.Git.Local/readme.md]: I have checked and corrected all files readme.md. Now everything is ok, all files are readable from webgui. I'm sorry for this bug.

File: Denis_Doubinin/03.Git.Hosting/readme.md (hunk 2 above)
Comment (DenisDoubinin) [reply Denis_Doubinin/03.Git.Hosting/readme.md]: I see that, but I can't understand why, my script mast takes varaibles one by one from 'git brunches' (its show only my brunches) and then checkout one by one to all brunches and  push them, but it takes to varaible b not only brunches that I can see after entering the command 'git brunch' but also all files from this brunches.

**Comment by DenisDoubinin** [comment]
I have checked and corrected all files readme.md. Now everything is ok, all files are readable from webgui. I'm sorry for this bug.

**Pull request closed without merging**